backend/app/Http/Controllers/EmployeeController.php

```diff
@@ -163,28 +163,30 @@ public function confirmCSV(Request $request)
     {
         $passed = 0;
         $failed = 0;
-        $file = json_decode($request->getContent(), true);
-        $json = $file['data'];
+        $file = json_decode($request->getContent(), true); 
+        $json = array_values(array_filter($file));
         $last_error = null;
 
         foreach ($json as $key => $item) {
             // only add employee where is_exist != true
-            if ($json[$key]['is_exist'] == false) {
+            // if ($json[$key]['is_exist'] == false) {
                 $raw_password = $this->generateRandomPwd(10);
                 $hash = Hash::make($raw_password);
-                unset($json[$key]['id']);
-                unset($json[$key]['is_exist']);
+                // unset($json[$key]['id']);
+                // unset($json[$key]['is_exist']); 
+                //...commented code will be removed after live testing...
+                unset($json[$key]['department']);
                 $json[$key]['id'] = Str::uuid();
                 $json[$key]['hash'] = $hash;
                 $json[$key]['raw_password'] = $raw_password;
 
                 $result = $this->insertEmployee($json[$key], $raw_password);
                 if (json_decode($result->getContent(), true)['errorState'] == true) $failed++;
                 else $passed++;
-                if (json_decode($result->getContent(), true)['error'] != null) $last_error = json_decode($result->getContent(), true)['error'];
-            }
+                if (json_decode($result->getContent(), true)['error'] != null) $last_error = json_decode($result->getContent(), true)['message'];
+            // }
```

Comment:
** the comments will be removed after testing on live. It was the previous implementation that worked.

I need the frontend to test my endpoint if he can consume it with his new methods, then i'll remove it. This one worked before for me, but I modified it because of the frontend

Reply:
Or should I remove it and keep somewhere else? That's also fine for me boss